aioesphomeapi/api.proto

```diff
@@ -283,7 +284,6 @@ message ListEntitiesCoverResponse {
   bool disabled_by_default = 9;
   string icon = 10;
   EntityCategory entity_category = 11;
-  bool supports_stop = 12;
```

Comment:
Perhaps this should be kept with `[deprecated = true]` or something

Reply:
Why are you removing this field? It was only just added recently

Reply:
I copied the api.proto from here: https://github.com/esphome/esphome/blob/dev/esphome/components/api/api.proto
and these were all the things that changed... I was clearly incorrect to think that keeping these two files in sync was a good idea.